gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions-snapshots/gradle-6.4-20200319134018+0000-bin.zip
+distributionUrl=https\://services.gradle.org/distributions-snapshots/gradle-6.4-20200401220055+0000-bin.zip
```

Comment:
Is this change intentional?

Reply:
Yes. There is an issue at the moments that the samples are tested against this ☝️ Gradle versions and not the current version under development. The previous wrapper version here did not contain JPMS support.